src/features/DebugSession.ts

```diff
@@ -51,10 +50,6 @@ export class DebugSessionFeature extends LanguageClientConsumer
         return new vscode.DebugAdapterInlineImplementation(debugAdapter);
     }
 
-    public dispose() {
-        this.command.dispose();
-    }
```

Comment:
Thanks for the contribution! Looks like due to the abstraction this class is inheriting, we'll need to leave `public dispose()` implemented (but empty):

```
src/features/DebugSession.ts(24,14): error TS2515: Non-abstract class 'DebugSessionFeature' does not implement inherited abstract member 'dispose' from class 'LanguageClientConsumer'.
```

Reply:
Done

Reply:
Well dang, that triggered:

> ERROR: 53:22  no-empty  block is empty

Are you able to build this locally? TSC should complain in the same way as CI here.

Reply:
You are right. I could either temporarily disable the tslint or add harmless variable:
```
/* tslint:disable:no-empty */
public dispose() {
}
/* tslint:enable:no-empty */
```
public dispose() {
    const noop = undefined;
}
```

Reply:
Sweet!

Reply:
Done. CI passed.